packages/azure/scripts/libcelo.sh

```diff
@@ -0,0 +1,9 @@
+get_secret () {
```

Comment:
What do you think about maybe brining downloading docker or mounting disks into here?

Reply:
I ended up bringing it into `provider-init.sh`: disk setup and initial pulling images is only done during initialization, but the "run scripts" use `get_secret` (and potentially other functions in libcelo.sh) every time they execute.